docs/transactions.md

```diff
@@ -1358,12 +1358,12 @@ Should be sent to trusted nodes only.
 
 **Status: Implemented**
 
-This transaction adds a NOC root certificate owned by the Vendor.
+This transaction adds a NOC root certificate (RCAC) owned by the Vendor.
```

Comment:
Can you please add clarifications to the section names as well, so that we have
#### ADD_NOC_ROOT (RCAC)
etc.